docs/operating-scylla/procedures/cluster-management/_common/membership-change-failures-note.rst

```diff
@@ -0,0 +1,7 @@
+.. note::
```

Comment:
Regarding commit ` doc: apply file includes, remove "Raft" `

Does this commit modify the changes done in previous commits?
If so, please don't. It's very confusing to the reviewers, I don't want to re-review the same parts again.

Instead, rebase and modify each commit in the series addressing the review feedback.

Commits (at least in this repository) should be constructed as if you knew all the review feedback before you started preparing the PR. The merged end result should have no trace that there was review.